pyproject.toml

```
@@ -1,3 +1,26 @@
+[project]
+name = "s2geometry"
+version = "0.11.0.dev1"
+description = "Python packaging of s2geometry"
```

use description from github repo
```suggestion
description = "Computational geometry and spatial indexing on the sphere"
```